### PR TITLE
tests/e2e: General fixes to the operator script

### PR DESCRIPTION
**Pull request by GabyCT**
This PR introduces general fixes to the test/e2e operator script, like fixing the indentation, using a variable that has been already declared and use in the script.

### CODE REVIEW

File: tests/e2e/operator.sh [hunk 1]
```diff
@@ -266,7 +265,7 @@ wait_for_stabilization() {
 	count=0
 	while true; do
 		change=0
-		pod_info=$(kubectl get pods -n confidential-containers-system -o=jsonpath='{range .items[*]}{.metadata.name}{" "}{range .status.containerStatuses[*]}{.name}{" "}{.restartCount}{"\n"}{end}{end}')
+		pod_info=$(kubectl get pods -n $op_ns -o=jsonpath='{range .items[*]}{.metadata.name}{" "}{range .status.containerStatuses[*]}{.name}{" "}{.restartCount}{"\n"}{end}{end}')
```
Comment (ldoktor) [inline tests/e2e/operator.sh:268]: To be sure it'd be better to quotate the `$op_ns` (just in case a space/whatever-weird-char is ever added)

**Review by ldoktor — COMMENTED** [review]
Thanks, unifying is always good (although I have to say I'm a 4-space person :-/). Please consider the quotation, even though it should not be needed it's a good practice to do so.

**Comment by GabyCT** [comment]
> Thanks, unifying is always good (although I have to say I'm a 4-space person :-/). Please consider the quotation, even though it should not be needed it's a good practice to do so.

@ldoktor thanks for the feedback and changes applied :)

**Comment by portersrc** [comment]
lgtm!

**Review by ldoktor — APPROVED** [review]
Thanks, lgtm

**Review by wainersm — APPROVED** [review]
Thanks @GabyCT !

**Comment by wainersm** [comment]
Let's see if https://github.com/confidential-containers/operator/pull/342 fixes the e2e-pr failures

**Comment by wainersm** [comment]
> Let's see if #342 fixes the e2e-pr failures

PR #342 is merged now. @GabyCT could you please rebase this PR so we can re-run the CI?

**Comment by GabyCT** [comment]
> > Let's see if #342 fixes the e2e-pr failures
> 
> PR #342 is merged now. @GabyCT could you please rebase this PR so we can re-run the CI?

@wainersm I just did the rebase :)

**Pull request merged**